aws/src/integration/java/org/apache/iceberg/aws/TestAssumeRoleAwsClientFactory.java

```diff
@@ -128,20 +130,15 @@ public void testAssumeRoleGlueCatalog() throws Exception {
     GlueCatalog glueCatalog = new GlueCatalog();
     assumeRoleProperties.put("warehouse", "s3://path");
     glueCatalog.initialize("test", assumeRoleProperties);
-    try {
-      glueCatalog.createNamespace(
-          Namespace.of("denied_" + UUID.randomUUID().toString().replace("-", "")));
-      Assert.fail("Access to Glue should be denied");
-    } catch (GlueException e) {
-      Assert.assertEquals(AccessDeniedException.class, e.getClass());
-    }
+    Assertions.assertThatThrownBy(
+            () ->
+                glueCatalog.createNamespace(
+                    Namespace.of("denied_" + UUID.randomUUID().toString().replace("-", ""))))
```

Comment:
In the future, it would be better to put the namespace in a variable to avoid such awkward auto formatting.